app/src/main/java/ch/epfl/sweng/studyup/question/AddQuestionActivity.java

```diff
@@ -86,28 +90,33 @@ public void addQuestion(View current) {
             RadioGroup answerGroup = findViewById(R.id.question_radio_group);
             RadioButton checkedButton = findViewById(answerGroup.getCheckedRadioButtonId());
             //get the tag of the button to know the answer number
-            int answerNumber = Integer.parseInt(checkedButton.getTag().toString());
+            int answerNumber = Integer.parseInt(checkedButton.getTag().toString()) - 1;
 
             boolean isTrueFalseQuestion = trueFalseRadioGroup.getCheckedRadioButtonId() == R.id.true_false_radio;
 
             //TODO: create the question, but first find a way to save them to the disk
-            String newQuestionFileID = UUID.randomUUID().toString();
+            String newQuestionFileID = UUID.randomUUID().toString() + ".png";
```

Comment:
In the future: maybe we could name the png with a certain format (section, year, difficulty infos for example)